Domains/4-Application/ISM/IntegratedStructuralModel.ecschema.xml

```diff
@@ -598,6 +609,7 @@
         </Target>
     </ECRelationshipClass>
 
+    <!-- DEPRECATED -->
```

Comment:
Same comment about using the Deprecated custom-attribute instead of just a comment in the XML.

Reply:
Fixed in https://github.com/iTwin/bis-schemas/pull/295/commits/f2ac98e614f8e110ebc9c4a5efbd309fb99ebea2